tests/e2e/test_vdbench_workload.py

```diff
@@ -248,7 +234,7 @@ def test_vdbench_workload(self, template, label_nodes, ripsaw, servers,
 
         log.info(sf_data)
 
-        timeout = 8640000  # 3600 (1H) * 24 (1D) * 100 = one hundred days
+        timeout = 86400  # 3600 (1H) * 24 (1D)  = one days
```

Comment:
@Avilir How long do we expect this test to run? If this test indeed runs for several hours, we may need to use a different marker (@performancelongrun?) and make use of this marker for long-running performance tests.

Reply:
i just remove the jenkins @workload marker, so it will not affect other tests.